modules/nf-core/ampcombi/meta.yml

```diff
@@ -0,0 +1,83 @@
+name: "ampcombi"
+description: A tool to parse and suumarise results from antimicorbial peptides tools and provide the respective functional classification.
+keywords:
+  - antimicrobial peptides
+  - amps
+  - parsing
+  - reporting
+  - align
+  - macrel
+  - amplify
+  - hmmsearch
+  - neubi
+  - ampir
+  - DRAMP
+tools:
+  - "ampcombi":
+      description: "This tool parses the results of amp prediction tools into a single table and aligns the hits against a reference database of antimicrobial peptides for functional classifications."
+      homepage: "https://github.com/Darcy220606/AMPcombi"
+      documentation: "https://github.com/Darcy220606/AMPcombi"
+      tool_dev_url: "https://github.com/Darcy220606/AMPcombi/tree/dev"
+      doi: ""
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - amp_input:
+      type: folder or filelist
```

Comment:
I don't think this fits with a schema.. .but we don't do much with this meta.yml so maybe can ignore.

Alternativeoly, why not two separate (but mutually exclusive) input channels, one for directory and one for the file list?